phoebe/frontend/bundle.py

```diff
@@ -12124,20 +12124,22 @@ def _scale_fluxes_cfit(fluxes, scale_factor):
                                 logger.debug("applying scale_factor={} to {} parameter in mesh".format(scale_factor, mesh_param.qualifier))
                                 mesh_param.set_value(mesh_param.get_value()*scale_factor, ignore_readonly=True)
 
-                # handle flux scaling based on distance and l3
-                # NOTE: this must happen AFTER dataset scaling
-                distance = self.get_value(qualifier='distance', context='system', unit=u.m, **_skip_filter_checks)
-                for flux_param in ml_params.filter(qualifier='fluxes', kind='lc', **_skip_filter_checks).to_list():
-                    dataset = flux_param.dataset
-                    if dataset in datasets_dsscaled:
-                        # then we already handle the scaling (including l3)
-                        # above in dataset-scaling
-                        continue
+                # alternate backends other than legacy already account for distance via pbflux
+                if computeparams.kind in ['phoebe', 'legacy']:
```

Comment:
so we don't need to do anything if the backend isn't one of these?

Reply:
phoebe and legacy were not affected by the bug, if that's what you're asking.  All other alternate backends which use pbflux were double-accounting for distance, but do not any longer.